src/ngx_http_lua_regex.c

```diff
@@ -2478,6 +2480,22 @@ ngx_http_lua_ffi_max_regex_cache_size(void)
     }
     return (uint32_t) lmcf->regex_cache_max_entries;
 }
+
+void
+ngx_http_lua_ffi_set_jit_stack_size(int size)
+{
+    int                          min_size;
+    ngx_http_lua_main_conf_t    *lmcf;
+
+    min_size = MIN(32 * 1024, size);
+    lmcf = ngx_http_cycle_get_module_main_conf(ngx_cycle,
+                                               ngx_http_lua_module);
+    if (lmcf == NULL) {
+        return;
+    }
+    pcre_jit_stack_free(lmcf->jit_stack);
+    lmcf->jit_stack = pcre_jit_stack_alloc(min_size, size);
```

Comment:
From my understanding of the docs there is no return code to check for errors http://pcre.org/original/doc/html/pcre_jit_stack_alloc.html